gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -408,6 +408,14 @@ public static Button createButton(Composite parent, String text, Listener listen
     return result;
   }
 
+  public static Button createButton(Composite parent, String text, int style, Color color, Listener listener) {
```

Comment:
Make the style the second argument after the parent to stick with the established pattern.